static/docs/commands-reference/metrics_modify.md

```diff
@@ -6,22 +6,20 @@ etc).
 ## Synopsis
 
 ```usage
-usage: dvc metrics modify [-h] [-q] [-v]
-                          [-t TYPE] [-x XPATH]
-                          path
+usage: dvc metrics modify [-h] [-q | -v] [-t TYPE] [-x XPATH] path
```

Comment:
keep multiple lines for readability?

Reply:
If fits without horizontal scroll like this:
![image](https://user-images.githubusercontent.com/1477535/59378018-04753600-8d19-11e9-98e3-9ffd4485cf30.png)
I thought that was the purpose of wrapping? The limit is around 72 chars BTW

Reply:
👍